src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -922,9 +920,7 @@ public static void addSignalHandler(final SignalHandler signalHandler) {
     }
 
     /**
-     * Inserts a low-cost Java safe-point in the code path.
-     * <p>
-     * Note: this does nothing on Java 8.
+     * Inserts a low-cost Java safe-point in the code path if -Djvm.safepoint.enabled
```

Comment:
we should also edit comment in https://github.com/OpenHFT/Chronicle-Core/blob/885750bdab180c35a6e61b8822b85eb7a537a9e5/src/main/java/net/openhft/chronicle/core/Jvm.java#L1536

Reply:
This has been checked to work on Java 8 update 202+, removed the comment